lib/src/flutter_range_slider.dart

```diff
@@ -469,7 +469,7 @@ class _RangeSliderState extends State<RangeSlider>
       showValueIndicator:
           sliderTheme.showValueIndicator ?? _defaultShowValueIndicator,
       valueIndicatorTextStyle: sliderTheme.valueIndicatorTextStyle ??
-          theme.textTheme.body2.copyWith(
+          theme.textTheme.bodyText1.copyWith(
```

Comment:
I think it should be bodyText2 as per the documentations. 
<img width="902" alt="Screenshot 2022-02-19 at 12 05 14 PM" src="https://user-images.githubusercontent.com/30897338/154789641-d81f1760-a05e-4408-9f87-96323b8a57a8.png">